package Student;

public class ExceptionCatcher {
	
	public void run() {
		
		boolean done = false;
		while (!done) {
			
			  InputChoice inputChoice = new InputChoice();
				
				try {
					
					inputChoice.returnChoice();	
					done = true;
				} catch (findbyidError e) {
			
					System.out.println(e.getMessage().toString());
					 inputChoice.returnChoice();
					 
					
				} catch (AgeErrorhandling e) {
					
					System.out.println(e.getMessage().toString());
					 inputChoice.returnChoice();	
					
				}
				
				catch (NameErrorHandling e) {
					
					System.out.println(e.getMessage().toString());
					 inputChoice.returnChoice();	
					
				}
				
				catch (Exception e) {
					System.out.println("Internal Error");
					inputChoice.returnChoice();
					
				}
			
		}
		
     
		
	}

}
